config/testdata/conf.good.yml

```diff
@@ -23,7 +23,7 @@ scrape_configs:
   # metrics_path defaults to '/metrics'
   # scheme defaults to 'http'.
 
-  file_sd_configs:
+  sd_files:
     - names:
```

Comment:
Yes I was just talking about `names` as I noticed that inconsistency while giving the workshop.